src/raw.rs

```diff
@@ -86,19 +91,25 @@ impl<W: Write> IntoRawMode for W {
         if set_terminal_attr(&mut ios as *mut _) != 0 {
             Err(io::Error::new(io::ErrorKind::Other, "Unable to set Termios attribute."))
         } else {
-            Ok(RawTerminal {
+            let mut res = RawTerminal {
                 prev_ios: prev_ios,
                 output: self,
-            })
+            };
+            try!(res.write(ENTER_MOUSE_SEQUENCE));
```

Comment:
I wonder: is this something we want to be on by-default? I think it would make more sense to have a RawTerminal newtype, which can be acquired through some consuming method enabling this mode. That'd be more "right".